tests/Framework/TestCase.php

```diff
@@ -49,6 +49,7 @@ public function getHttpRequest()
      *
      * @param string $path
      * @param array<string, string> $substitutions
+     * @psalm-suppress ImplementedReturnTypeMismatch
```

Comment:
can you provide a comment on the end of this line saying why?

Reply:
ditto